po/da.po

```diff
@@ -960,9 +956,8 @@ msgid "Async Traits"
 msgstr "Asynkrone egenskaber (eng. Traits)"
 
 #: src/SUMMARY.md:297 src/async/pitfalls/cancellation.md:1
-#, fuzzy
 msgid "Cancellation"
-msgstr "Installation"
+msgstr "Afbrydelser"
```

Comment:
nit: Cancellation is singular, Afbrydelser is plural.

Reply:
Hmm, good point... perhaps _Annulering_ is a better translation?

```suggestion
msgstr "Afbrydelser"
```